index.html

```diff
@@ -26,6 +40,9 @@
         <span class="info">ex. Game, Music, Video, Photography</span>
       </form>
     </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
Sorry, ignore the styling changes here! My editor formats on save so it looks like it broke some of these elements into their own lines. The big change here is I imported jquery and the fetch.js script